proxy/logging/LogAccess.cc

```diff
@@ -1782,6 +1782,40 @@ LogAccess::marshal_client_req_protocol_version(char *buf)
   return len;
 }
 
+/*-------------------------------------------------------------------------
+  -------------------------------------------------------------------------*/
+
+int
+LogAccess::marshal_server_req_protocol_version(char *buf)
```

Comment:
Not a big deal, but I think this part is reviewable even if a reviewer is not so familiar with H2 and server session. I'd be happy to add the new log field beforehand as a preparatory step.

Reply:
Sure we could land a very simple version of this first.

Reply:
Added PR #7680